app/src/main/java/com/hedvig/app/feature/home/ui/HomeAdapter.kt

```diff
@@ -310,6 +317,41 @@ class HomeAdapter(
             )
         }
 
+        class HowClaimsWorkButton(parent: ViewGroup) :
+            ViewHolder(parent.inflate(R.layout.how_claims_work_button)) {
+            private val binding by viewBinding(HowClaimsWorkButtonBinding::bind)
+            override fun bind(
+                data: HomeModel,
+                fragmentManager: FragmentManager,
+                retry: () -> Unit,
+                requestBuilder: RequestBuilder<PictureDrawable>,
+                tracker: HomeTracker
+            ): Any? = with(binding) {
+                if (data !is HowClaimsWork) {
+                    return invalid(data)
+                }
+                val howClaimsWorkData = data.pages.mapIndexed { index, page ->
+                    if (index == data.pages.size - 1) {
```

Comment:
Move this if to the row where we decide which button text we send to the `NoTitlePage`